server/FakeToOriginalIPConverter/README.md

```diff
@@ -18,7 +18,11 @@ It updates the TcpClient/TcpServer address and port details based on the informa
             "release": "0.0.1",
             "fake-tcp-ip": "1.1.3.1",
             "fake-tcp-port": 3001,
-            "original-tcp-ip": "127.0.0.1",
+            "original-tcp-ip": {
```

Comment:
Can we change naming original-tcp-ip to original-address and fake-tcp-ip to fake-address and make appropriate changes in code as well because we might have either ip-address or domain-name felt that would be better